domains/transfer/Transfer.API/API/Transfer/Api/v2024_01_03/Dto/Responses/TransferAutomationStatus.cs

```diff
@@ -0,0 +1,3 @@
+namespace API.Transfer.Api.v2024_01_03.Dto.Responses;
+
+public record TransferAutomationStatus(bool Healthy);
```

Comment:
This looks like some legaacy and can be removed

Reply:
okay

Reply:
fixed here:

https://github.com/Energinet-DataHub/energy-origin/pull/558/commits/0ea9743cb222e5b66335ac4ed2e7677c66104832